api/src/main/java/com/exadel/frs/core/trainservice/entity/Face.java

```diff
@@ -33,7 +30,7 @@
     private String faceName;
     @Column(name = "api_key")
     private String apiKey;
-    @Type(type = "jsonb")
+    @Type(type = "json")
```

Comment:
While adding a new face now:

Exception: Caused by: org.postgresql.util.PSQLException: ERROR: column "embeddings" is of type jsonb but expression is of type character varying